### PR TITLE
Project UI

### PR DESCRIPTION
**Pull request by ericmahare**
In this section, I did the following: -

- [ ] Created a method to List all books.
- [ ] Created a method to List all people.
- [ ] Created a person method to add new student or teacher
- [ ] Create a book method for adding a new book
- [ ] Created a rental method for adding new library rental
- [ ] Created a method to List all rentals for a given person's id.

### CODE REVIEW

File: app.rb [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+require_relative('./student')
+require_relative('./teacher')
+require_relative('./books')
+require_relative('./rental')
+
+class App
+  def initialize
+    @people = []
+    @books = []
+  end
```
Comment (EricMbouwe) [inline app.rb]: - [ ] You are missing the rental list in your constructor.
```rb
def initialize
    @books = []
    @people = []
    @rentals = []
end
```

File: app.rb [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+    until [1, 2].include?(entry)
+      entry = gets.chomp.strip.to_i
+      puts 'Choice is not correct : (1) for Student, (2) for Teacher' unless [1, 2].include?(entry)
+    end
+    case entry
+    when 1
+      create_student
+    when 2
+      create_teacher
+    end
```
Comment (EricMbouwe) [inline app.rb]: - [ ] Let's add a  catch for user bad entries so that the app doesn't crash if a user enters a different value than 1 or 2.

**Review by EricMbouwe — CHANGES_REQUESTED** [review]
Hi @ericmahare, 

Good job so far 🔥!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights 🔥🎉
- ✔ Linters
- ✔  PR
- ✔  Class methods

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by omar-muhamad — APPROVED** [review]
# Approved ✔
### Hi @ericmahare,

Great job, Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

thumbs-up-agree


## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- Github flow is used correctly. ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- N/A

## [Optional] suggestions:

- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**